units/en/unitbonus3/introduction.mdx

```diff
@@ -1,11 +1,64 @@
-# Introduction
+# DIAMBRA Arena Overview
 
```

Comment:
I would add an introduction.

Welcome to this new bonus unit where you'll learn to use DIambra and train agents to play 

At the end of the unit you'll get

illustreation of agent playing